code/modules/jobs/job_types/_job.dm

```diff
@@ -65,6 +65,19 @@
 
 	var/bounty_types = CIV_JOB_BASIC
 
+/datum/job/New()
+	var/list/jobs_changes = GetMapChanges()
```

Comment:
```suggestion
/datum/job/New()
	. = ..()
	var/list/jobs_changes = GetMapChanges()
```
all job datum new procs call parent
no particular reason yet, in case you ask